### ISSUE TITLE
Image built from scratch fails due to an unrecognized option in gitlab-ci

### ISSUE THREAD

**Issue by Porkepix** [koalaman/shellcheck#1795]
#### For bugs
- My shellcheck version (`shellcheck --version` or "online"):  docker image on latest version.

#### For new checks and feature suggestions
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

The gitlab-ci relevant part is the following one:
```sh
shellcheck:
  image: koalaman/shellcheck:latest
  stage: validate
  script:
    - shellcheck ansible_bootstrap.sh
```
This is basically the exact same one as visible on the wiki here: https://github.com/koalaman/shellcheck/wiki/GitLab-CI

The result is that it completely fail as it looks like it's receiving arguments it shouldn't (a `-c`). I tried to tinker around it, but no way to have it working other than having the exact same step on the .gitlab-ci file but with the `shellcheck-alpine` image: this one works, but the image with only the binary should be enough.

Below is the output of Gitlab-CI:

```sh
Running with gitlab-runner 12.7.0-rc1 (4314e2d9)
  on docker-auto-scale fa6cab46
Using Docker executor with image koalaman/shellcheck:latest ...
00:09
Pulling docker image koalaman/shellcheck:latest ...
Using docker image sha256:ecb598547d3d0726dc3ebc9ad0cff4474653bfbb46e43d82f3cf5b76f64b41a9 for koalaman/shellcheck:latest ...
Running on runner-fa6cab46-project-16178920-concurrent-0 via runner-fa6cab46-srm-1579016540-4c5008d7...
00:01
$ eval "$CI_PRE_CLONE_SCRIPT"
00:02
Fetching changes with git depth set to 50...

[…]

Checking out 495f08c5 as reproduce_shellcheck_image_issue...
Skipping Git submodules setup
unrecognized option `-c'
00:01
Usage: shellcheck [OPTIONS...] FILES...

[…]

ERROR: Job failed: exit code 3
```

#### Here's what I wanted or expected to see:

The `shellcheck`simple image should be enough and therefore shouldn't produce any issue, especially by using an almost copy & paste of the wiki example, which is less than one month old.

**Comment by carlbordum** [comment]
Hi @Porkepix,

I was just boxing with the same issue. The problem is that we need to use the `koalaman/shellcheck-alpine:latest` image. I have updated the wiki (https://github.com/koalaman/shellcheck/wiki/GitLab-CI) and added one more (advanced) example. 

I hope it works for you,
Carl

**Comment by Porkepix** [comment]
Yes, that's what we did. But I think it should somehow be possible to use the simple image containing only the binary and nothing more, right?

**Comment by SvenMW** [comment]
this still seems broken, at least in gitlab CI.

I still get the error `unrecognized option `-c'` but I didn't pass this information to the shellcheck binary.

I also don't think the "fix" would be to use the alpine image, but rather figure out, what is wrong with the binary only image and fix that.

I will try to investigate myself, but can't make any promises.
